### ISSUE TITLE
MFILE does not output PF inductance matrix

### ISSUE THREAD

**Issue by jonmaddock** [ukaea/PROCESS#1282]
In GitLab by @apetrov on Mar 15, 2021, 10:55

We can see the PF inductance matrix in the OUT file but they don't appear in MFILEs. This makes it challenging to export inductances for further analysis.

![pfmissing](https://git.ccfe.ac.uk/process/process/uploads/39a7ce0ba48e95a9dae41749e26163df/pfmissing.JPG)

Not too sure if this is a bug or conscious decision - just noticed this while working on Stage 1b.

**Comment by chris-ashe** [comment]
Agreed with @timothy-nunn that this should be left until the data structure refactor with the new MFILE
